packages/fxa-payments-server/src/App.tsx

```diff
@@ -55,39 +58,48 @@ export const App = ({
   FlowEvents.init(queryParams);
   return (
     <AppContext.Provider value={appContextValue}>
-      <AppErrorBoundary>
-        <StripeProvider apiKey={config.stripe.apiKey}>
-          <ReduxProvider store={store}>
-            <Router>
-              <React.Suspense fallback={<RouteFallback />}>
-                {/* Note: every Route below should also be listed in INDEX_ROUTES in server/lib/server.js */}
-                <Route
-                  path="/"
-                  exact
-                  render={() => <Redirect to="/subscriptions" />}
-                />
-                <Route
-                  path="/subscriptions"
-                  exact
-                  render={props => (
-                    <SettingsLayout>
-                      <Subscriptions {...props} />
-                    </SettingsLayout>
-                  )}
-                />
-                <Route
-                  path="/products/:productId"
-                  render={props => (
-                    <SignInLayout>
-                      <Product {...props} />
-                    </SignInLayout>
-                  )}
-                />
-              </React.Suspense>
-            </Router>
-          </ReduxProvider>
-        </StripeProvider>
-      </AppErrorBoundary>
+      <AppLocalizationProvider
+        userLocales={navigator.languages}
+        bundles={['main']}
+      >
+        <Localized id="document" attrs={{ title: true }}>
+          <DocumentTitle title="fIREFOx aCCOUNTs pAYMENTs">
```

Comment:
Is this here so that you can ensure that fluent overrides the value?

Reply:
Correct, this is the method we used through out https://github.com/mozilla-lockwise/lockwise-addon

Reply:
Removed this formatting